scripts/map-randomizer.js

```diff
@@ -68,19 +69,24 @@
     shuffleArray(available);
     const selection = available.slice(0, amount);
 
-    if (!selection.length) return target.innerHTML = `
-          <div class="empty">
-            <div class="empty-icon">
-              <span class="icon icon-emoji"></span>
-            </div>
-            <p class="empty-title h5">Congratulations</p>
-            <p class="empty-subtitle">You've successfully selected no maps, you muppet</p>
+    if (!selection.length) {
+      target.innerHTML =
+        `<div class="empty">
+          <div class="empty-icon">
+            <span class="icon icon-emoji"></span>
           </div>
-        `;
+        <p class="empty-title h5">Congratulations</p>
+        <p class="empty-subtitle">You've successfully selected no maps, you muppet</p>
+        </div>`;
+      return;
+    }
 
-    target.innerHTML = '<ol>' + selection.map(function format(map) {
-      return `<li><span class="chip">${map.name}</span></li>`;
-    }).join('') + '</ol>';
+    target.innerHTML = selection.map((entry, idx) => {
+      return `<div class="img-container">
```

Comment:
While adding images gives nice visuals, it should not make it harder to use the site. In this case, we're adding so much visuals that when we're presented with 3 maps, you are most likely going to need to scroll just in order to see the map names. 

I would recommend maybe using the images as backgrounds and give the map summary a fixed height so when people come to the site for maps, they can see all options at a glance.

Reply:
I understand your point, but from all my examples, scrolling has been unnecessary on wider screens. On mobile, you have to scroll, yes, but the alternative is very small icons that are near illegible.

Could you elaborate?

Reply:
For example:
3 maps chosen:
![image](https://user-images.githubusercontent.com/5606617/135038618-96074e15-768a-4af7-ac42-7dd9184c15e8.png)

4 maps chosen:
![image](https://user-images.githubusercontent.com/5606617/135038695-73dc4e76-f997-4204-981d-7f50dd1cc723.png)

Reply:
> On mobile, you have to scroll, yes, but the alternative is very small icons that are near illegible.

Which was the main reason why this was build in the first place as not everyone is going to have access to a fully functioning PC. Mobile should not never be an after thought. In order for this functionality to land it needs to be solved correctly. In this case I would suggest simply not rending the image on mobile devices but just present the list as normally and then progressively enhance the layout for larger form factors.

Reply:
I see. though I'm wondering if that actually is an issue. Scrolling on a mobile is much more natural and reactive than on PC. That being said, I'll remove the rendering on mobile, and if there's a request for rendered images, well then it shouldn't be much work to add it back in.

Reply:
Thanks for the feedback though! I've had so much fun learning simple web-dev that I completely forgot why this was built in the first place!

Reply:
@SahandJ Haha, don't worry, I completely understand it. It's often easy to get carried doing something you enjoy.